salt/modules/win_system.py

```diff
@@ -146,35 +145,36 @@ def reboot(timeout=5, in_seconds=False, wait_for_reboot=False,  # pylint: disabl
     '''
     Reboot a running system.
 
-    :param int timeout:
-        Number of minutes/seconds before rebooting the system. Minutes vs
-        seconds depends on the value of ``in_seconds``.
-        Default is 5 minutes.
-
-    :param bool in_seconds:
-        Whether to treat timeout as seconds or minutes.
+    Args:
 
-        .. versionadded:: 2015.8.0
+        timeout (int):
+            Number of minutes/seconds before rebooting the system. Minutes vs
+            seconds depends on the value of ``in_seconds``. Default is 5
+            minutes.
 
-    :param bool wait_for_reboot:
+        in_seconds (bool):
+            Whether to treat timeout as seconds or minutes.
 
-        Sleeps for timeout + 30 seconds after reboot has been initiated.
-        This is useful for use in a highstate for example where
-        you have many states that could be ran after this one. Which you don't want
-        to start until after the restart i.e You could end up with a half finished state.
+            .. versionadded:: 2015.8.0
 
-        .. versionadded:: 2015.8.0
+        wait_for_reboot (bool)
+            Sleeps for timeout + 30 seconds after reboot has been initiated.
+            This is useful for use in a highstate for example where you have
+            many states that could be ran after this one. Which you don't want
```

Comment:
These are sentence fragments. We likely need to connect `this one` to `Which` with a comma.